packages/rpc/src/parsers.ts

```diff
@@ -15,30 +15,30 @@ const chainAssetMapFactory = <Z extends z.ZodTypeAny>(parser: Z, defaultValue: z
     Bitcoin: z.object({ BTC: parser }),
     Ethereum: z.object({ ETH: parser, USDC: parser, FLIP: parser, USDT: parser }),
     Polkadot: z.object({ DOT: parser }),
-    Arbitrum: z
```

Comment:
would this break if we use it on older blocks?

Reply:
Yes but for that you can use the older package